docs/articles/documentation/test-api/selecting-page-elements/examples-of-working-with-dom-elements.md

```diff
@@ -81,7 +102,49 @@ test('My test', async t => {
 });
 ```
 
-## Checking if an Element is Available
+## Access Child Nodes in the DOM Hierarchy
+
+The selector API includes methods that allow you to [search for elements in the DOM tree](selectors/functional-style-selectors.md#search-for-elements-in-the-dom-hierarchy). For instance, you can use the [child](selectors/functional-style-selectors.md#child), [parent](selectors/functional-style-selectors.md#parent), and [sibling](selectors/functional-style-selectors.md#sibling) methods to traverse up, down or aside through the hierarchy.
+
+However, these methods have a limitation. They iterate DOM elements only and ignore other node types. To get a child text node for a selector, use a function as shown in this example.
+
+Consider the following *example.html* page:
+
+```js
+<!DOCTYPE html>
+<html>
+    <body id="testedpage">
+        This is my tested page. <!--This is the first child node of <body>-->
+        <p>My first paragraph.</p>
+        <p>My second paragraph.</p>
+    </body>
+</html>
+```
+
+Assume you need to verify text content of the first node in the `<body>` (*'This is my tested page'*).
+
+First, create a selector that identifies the `testedpage` element. Then pass this selector as a [dependency](selectors/selector-options.md#optionsdependencies) to another selector that returns the [childNodes](https://developer.mozilla.org/en-US/docs/Web/API/Node/childNodes) collection.
+
+```js
+import { Selector } from 'testcafe';
+import { ClientFunction } from 'testcafe';
+
+fixture `My Fixture`
+    .page `example.html`;
+
+    const testedPage = Selector('#testedpage');
+
+    const childNodes = Selector(() => {
```

Comment:
We can also use `find` method and it could be even easier `Selector('#testedpage').find(node => node)`

Reply:
Well, I guess it doesn't do exactly the same (according to the docs, `childNodes` returns only immediate children, while `find` returns all the descendants). Besides, `.find(node => node)` doesn't look like it makes a lot of sense.